css/properties/display.json

```diff
@@ -871,6 +1071,279 @@
                 "deprecated": false
               }
             }
+          },
+          "xul_box_values": {
+            "__compat": {
+              "description": "<code>-moz-box</code> and <code>-moz-inline-box</code>",
+              "support": {
+                "chrome": {
+                  "version_added": false
+                },
+                "chrome_android": {
+                  "version_added": false
+                },
+                "edge": {
+                  "version_added": false
+                },
+                "edge_mobile": {
+                  "version_added": false
+                },
+                "firefox": {
+                  "version_added": true,
+                  "notes": "Scheduled for removal (See <a href='https://bugzil.la/879275>bug 879275)"
```

Comment:
Notes should end with a full stop and the link should have proper markup.
`"Scheduled for removal (see <a href='https://bugzil.la/879275'>bug 879275</a>)."`